Automattic/wp-calypso
# Command palette: Tweaks for new nav redesign

Related to https://github.com/Automattic/dotcom-forge/issues/7640

## Proposed Changes

 * Open hosting configuration (=> update the name to Open server settings)
 * Open PHP logs (=> /site-logs/:site/php)
 * Open web server logs (=> /site-logs/:site/web)
 * Change admin interface style (=> /settings/general/:site or options-general.php)
 * Remove admin interface alias from hosting config
 * Manage staging sites (=> /staging-site/:site)

## Why are these changes being made?


To go directly to the proper locations in the new navigation

## Testing Instructions



* Use calypso live
* Use ⌘k to launch the command palette
* Try the aforementioned commands, ensure they take you to the right and proper place as mentioned above.

## Pre-merge Checklist
